dGame/dMission/RacingTaskParam.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <cstdint>
+
+enum class RacingTaskParam : int64_t {
+    RACING_TASK_PARAM_FINISH_WITH_PLACMENT = 1,             //<! A task param for finishing with a specific placement.
```

Comment:
Looks like there is a typo (first 'e' of 'placement' is missing): <code>RACING_TASK_PARAM_FINISH_WITH_PLAC<ins>E</ins>MENT</code>

Reply:
Fixed.  Good spot!